tensorflow/contrib/verbs/rdma.cc

```diff
@@ -793,9 +841,16 @@ void RdmaTensorBuffer::PostCopyOperations(
       VLOG(2) << "Extend RDMA buffer from " << size_ << " to " << buffer_size;
     }
     CreateCPUBuffer(buffer_size, false);
+    //Need to be recieved again, put into the re-recv queue and the table
+    requeue.push(key_with_step_id);
+    ReItem* item = new ReItem();
+    item->send_args = send_args;
+    item->recv_args = recv_args;
+    item->in = in;
```

Comment:
Do we need to increase reference count on [this](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/rendezvous.cc#L217-L219) or [that](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/rendezvous.cc#L175-L177)?
```
      if (item->recv_args.device_context) {
        item->recv_args.device_context->Ref();
      }
```